enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -52,7 +52,7 @@ <h2 class="course-confirmation-title">{{ confirmation_text }}</h2>
               <i class="fa fa-clock-o" aria-hidden="true"></i>
               <span>{{ starts_at_text }} {{ course_start_date }} &nbsp;| &nbsp; {{ course_pacing }}</span>
             </div>
-            {{ course_short_description }}
+            {{ course_short_description | striptags }}
```

Comment:
Were you able to produce the issue locally? And is this fix working as expected?

Reply:
Yes, I added `<p>` tags in my course to locally test it and the fix is working.